README.md

```diff
@@ -1,3 +1,7 @@
+Current Build Status
+--------------------
+
+[![Build Status](https://travis-ci.org/leider/CoolBeans.png)](https://travis-ci.org/leider/CoolBeans)
```

Comment:
I've not used Travis before. I see that this references your fork. Should this be updated to reference my repo?